host.go

```diff
@@ -148,12 +146,12 @@ func (h *gpbftRunner) Start(ctx context.Context) (_err error) {
 			// Handle messages, finality certificates, and alarms
 			select {
 			case c := <-finalityCertificates:
-				if err := h.receiveCertificate(c); err != nil {
-					return err
-				}
+				h.receiveCertificate(c)
 			case <-h.alertTimer.C:
 				if err := h.participant.ReceiveAlarm(); err != nil {
-					return err
+					// TODO: Probably want to just abort the instance and wait
+					// for a finality certificate at this point?
```

Comment:
I think it's also fine to do what we're doing here. Ignore the failure and keep receiving messages. I think? I'm not sure.